proxy/proxy-impl/src/test/java/org/apache/aries/blueprint/proxy/ProxySubclassGeneratorTest.java

```diff
@@ -173,6 +174,7 @@ public void testFinalClass() throws Exception
    * Test a private constructor
    */
   @Test
+  @Ignore
```

Comment:
I haven't found good replacement for sun.reflect.* to make private constructor accessible

Reply:
objenesis seems to be good replacement